### PR TITLE
sepia-fog-images: Run automatically every week

### PR DESCRIPTION
**Pull request by djgalloway**
Signed-off-by: David Galloway <dgallowa@redhat.com>

### CODE REVIEW

File: sepia-fog-images/config/definitions/sepia-fog-images.yml [hunk 1]
```diff
@@ -15,10 +15,14 @@
...
 
+    # Run every Sunday at noon
+    triggers:
+      - timed: "0 12 * * 0"
+
     parameters:
       - string:
           name: DISTROS
-          default: "ubuntu_14.04 ubuntu_16.04 centos_7.5 rhel_7.5"
+          default: "ubuntu_18.04 centos_8.1 rhel_8.1"
```
Comment (toabctl) [inline sepia-fog-images/config/definitions/sepia-fog-images.yml]: Can we have that for openSUSE images, too?

Comment (djgalloway) [reply sepia-fog-images/config/definitions/sepia-fog-images.yml]: I need some help from Suse with the autoinstallation.  It works from start to finish automatically on 15.1 but 15.2 stops in the installation and asks for the URL of the ISO repo even though it is passed as a kernel option.

Comment (toabctl) [reply sepia-fog-images/config/definitions/sepia-fog-images.yml]: are you using autoyast? is there more information what you are doing?

Comment (djgalloway) [reply sepia-fog-images/config/definitions/sepia-fog-images.yml]: Sorry, yes, I started to gather this info and got pulled away.

Here is our autoyast XML: https://github.com/ceph/ceph-cm-ansible/blob/master/roles/cobbler/templates/kickstarts/cephlab_opensuse_leap.xml

Here is what is being passed as a kernel parameter to the installation: `install=http://172.21.0.11/cblr/links/openSUSE-15.1-x86_64/` (of course the URL changes to 15.2 if the distro is 15.2)

Installation completes successfully in 15.1 but the installer stops and asks for the link above during 15.2 autoinstall.

Comment (toabctl) [reply sepia-fog-images/config/definitions/sepia-fog-images.yml]: Is there anything in the logs maybe? Or is it possible to access the system?

Comment (djgalloway) [reply sepia-fog-images/config/definitions/sepia-fog-images.yml]: As far as I can tell, there is no way to check logs from within YaST.

On 15.1, the installer runs from start to finish.  On 15.2, it stops at `please make sure your installation media is available`

Comment (djgalloway) [reply sepia-fog-images/config/definitions/sepia-fog-images.yml]: You're welcome to ssh to smithi166 and dig through `/var/log/YaST2`

Comment (toabctl) [reply sepia-fog-images/config/definitions/sepia-fog-images.yml:25]: I'll try to have a look. sorry for not responding anymore....

**Pull request merged**